### ISSUE TITLE
installation fails with untar error

### ISSUE THREAD

**Issue by jheeffer** [Gapminder/ddf-validation#294]
Platform: win10
npm: 4.0.3
node: 6.9.2

Tried: 
- run as administrator
- `npm cache clean`
- installing other packages: `npm i vizabi -g` does not fail
- `npm install ddf-validation@1.1.5 -g` does not fail

```
C:\Users\Jasper Heeffer>npm i ddf-validation -g
npm ERR! tar.unpack untar error C:\Users\Jasper Heeffer\AppData\Roaming\npm-cache\ddf-validation\1.2.1\package.tgz
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\Jasper Heeffer\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "i" "ddf-validation" "-g"
npm ERR! node v6.9.2
npm ERR! npm  v4.0.3
npm ERR! path C:\Users\Jasper Heeffer\AppData\Roaming\npm\node_modules\.staging\ddf-validation-f207c00f\test\fixtures\rules-cases\incorrect-file\on.csvint-chars\ddf--entities--regi
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall open

npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\Jasper Heeffer\AppData\Roaming\npm\node_modules\.staging\ddf-validation-f207c00on.csv'fixtures\rules-cases\incorrect-file\non-print-chars\ddf--entities--regi
npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\Jasper Heeffer\AppData\Roaming\npm\node_modules\.staging\ddf-validation-f207c00on.csv'fixtures\rules-cases\incorrect-file\non-print-chars\ddf--entities--regi
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Jasper Heeffer\npm-debug.log
```

**Comment by buchslava** [comment]
@jheeffer just fixed try version `1.2.3`

**Comment by jheeffer** [comment]
Couldn't reproduce now, so fixed : )
